spec/models/rules_of_origin/scheme_set_spec.rb

```diff
@@ -99,14 +99,6 @@
     end
   end
 
-  describe '#links' do
```

Comment:
So my understanding is that we still have a links method but in our uk schemes file this is now empty?

Do we want to remove the corresponding method since it's no longer in use or have I misunderstood this?

Reply:
@willfish Only one of the links that is at 'UK' level was removed from the UK Backend json file but there are other links that are iterated over in the page (that are FTA specific) so we still need the method.

Hope this makes sense. :) 

![image](https://github.com/trade-tariff/trade-tariff-backend/assets/127106895/8047a05a-46ff-4c26-b527-263e860d5453)

Reply:
Ah okay, so we have links for specific schemes in the scheme set and a links attribute of the scheme set itself. I'm guessing that these are all merged together at some point. I wonder if it's worth keeping the test and just saying its empty?

```  describe '#links' do
    subject { scheme_set.links }

    it { is_expected.to eq [] }
  end
```

Reply:
Not sure if there's value to that, tbh